rst_files/mccall_model_with_separation.rst

```diff
@@ -274,6 +270,19 @@ Let's plot the approximate solutions :math:`U` and :math:`V` to see what they lo
 
 We'll use the default parameterizations found in the code above
 
+.. code-block:: julia
+
+    # a default utility function
+    u(c, σ) = (c^(1 - σ) - 1) / (1 - σ) 
+
+    # default objects
+    const n = 60                                           # n possible outcomes for wage
+    const default_w_vec = range(10, 20, length = n)        # wages between 10 and 20
+    const a, b = 600, 400                                  # shape parameters
+    const dist = BetaBinomial(n-1, a, b)                   # wage distribution
+
```

Comment:
The these globals aren't really needed at all. Best to drop these.

Reply:
Yeah, the style guide had no `const` as well

Reply:
Done.